src/icons/icons.yml

```diff
@@ -7889,6 +7889,14 @@ icons:
     categories:
       - Brand Icons
 
+  - name:       Vue.js
+    id:         vue-js
```

Comment:
let us use an `id` that is requested in the icon request issue, i.e, `fa-vue` / `fa-vuejs`. Maybe we could also set up an alias for both.

Reply:
Since the name of the lib is "vue.JS" I chose `fa-vuejs`.

Reply:
If you would like, you could add an `aliases` key for the icon in `src/icons/icons.yml`, but that is not a requirement though.